docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -104,16 +104,19 @@ The following table illustrates the {fleet} user actions available to different
 
 To manage your {agent}s and the data they collect, create a new policy:
 
-. Log in to {kib} and go to *Management* > *{fleet}*.
+. In {kib}, go to **Management -> {fleet}**.
 
-. In {fleet}, click *Agent policies* > *Create agent policy*.
+. In {fleet}, click **Agent policies -> Create agent policy**.
 Name your policy. All other fields are optional and can be modified later.
 By default, each policy enables the _system_ integration, which collects system information and metrics.
 +
 [role="screenshot"]
 image::images/create-agent-policy.png[{fleet} in {kib}]
-
-. Click *Create agent policy*.
++
+. Create the agent policy:
+* To use the UI, click **Create agent policy**.
+* beta[] To use the {fleet} API, click **Preview API request** and run the
+request.
```

Comment:
What do you think of making it more explicit that the reader should choose one of the two options? (I know you are indicating that with the unordered list, but it might be worth reinforcing that with words, too.) I'm not sure what the right words are, but maybe something like... 

```suggestion
. Create the agent policy. There are two ways to choose from:
* To use the UI, click **Create agent policy**.
* beta[] To use the {fleet} API, click **Preview API request** and run the
request.
```

Reply:
Hmm. After reading the rest of the diff, I'm not sure if this is a good suggestion, but I'll leave it here for you to decide. 🙂

Reply:
@colleenmcginnis Generally it's best to avoid introducing a list with the number of items that it contains, but I could say, "Do one of the following." I like to be a minimalist if something seems obvious, though. WDYT?

Reply:
>Generally it's best to avoid introducing a list with the number of items that it contains

Ah yes, that makes sense. I'm fine with either adding "Do one of the following" or leaving it as is -- whichever you feel is better.

Reply:
At this point, I will err on the side of lazy and leave it as-is. :-)  We can always clean it up later.